### PR TITLE
Add tagger tags to pod manifests

### PR DESCRIPTION
**Pull request by aureleoules**
### What does this PR do?

The manifest payloads have already had a tags field added to the payloads - but we need to add all the tags to the individual manifest the agent would normally send to only the orch track to the orchmanif track too.

I tested that it works on my local minikube cluster.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -642,7 +642,7 @@ require (
 
 require (
 	github.com/DATA-DOG/go-sqlmock v1.5.2
-	github.com/DataDog/agent-payload/v5 v5.0.137
+	github.com/DataDog/agent-payload/v5 v5.0.138-0.20241128131042-e8f5da68c088
```
Comment (aureleoules) [inline go.mod:645]: will lock version when https://github.com/DataDog/agent-payload/pull/355 is merged

**Review by kangyili — APPROVED** [review]

**Review by kayayarai — APPROVED** [review]

File: pkg/collector/corechecks/cluster/orchestrator/processors/k8s/pod.go [hunk 2]
```diff
@@ -164,6 +164,19 @@ func (h *PodHandlers) ResourceVersion(ctx processors.ProcessorContext, resource,
 	return resourceModel.(*model.Pod).Metadata.ResourceVersion
 }
 
+// ResourceTaggerTags is a handler called to retrieve tags for a resource from the tagger.
+//
+//nolint:revive // TODO(CAPP) Fix revive linter
+func (h *PodHandlers) ResourceTaggerTags(ctx processors.ProcessorContext, resource interface{}) []string {
+	r := resource.(*corev1.Pod)
```
Comment (remeh) [inline pkg/collector/corechecks/cluster/orchestrator/processors/k8s/pod.go:171]: You should consider grabbing the second parameter returned from a type assertion to validate it worked.

**Review by mackjmr — APPROVED** [review]

**Review by kkhor-datadog — APPROVED** [review]

**Review by remeh — APPROVED** [review]

**Review by KevinFairise2 — APPROVED** [review]

**Pull request merged**